docs/upgrade.html

```diff
@@ -17,16 +17,14 @@
 
 <script><!--#include virtual="js/templateData.js" --></script>
 
-<script id="upgrade-template" type="text/x-handlebars-template">
```

Comment:
I will add back this javascript template in the next commit. Guozhang just told me that we are expected to have a local webserver in order to read this upgrade.html.